### ISSUE TITLE
Update MoveRadioTelescopeByXDegrees() to check for software-stops

### ISSUE THREAD

**Issue by tfranks31** [YCPRadioTelescope/YCP-RT-ControlRoom#453]
`MoveRadioTelescopeByXDegrees()` was not implemented when the software-stops branch was created, but it is implemented now in Development. This method needs to be updated to check if the elevation is past the software-stops limit and if it is, don't allow any movement going further past the position.

This change can be made once software-stops is merged into Development.
